packages/generator-single-spa/src/react/generator-single-spa-react.js

```diff
@@ -112,6 +130,28 @@ module.exports = class SingleSpaReactGenerator extends Generator {
 
   }
   install() {
+    if (this.typescript) {
+      const typescriptDevDeps = [
+        "typescript",
+        "fork-ts-checker-webpack-plugin",
+        "@babel/preset-typescript",
+        "eslint-config-ts-react-important-stuff",
+      ]
+      const typeDeps = [
+        "@types/jest",
+        "@types/react",
+        "@types/react-dom",
+        "@types/systemjs",
+        "@types/webpack-env",
+      ]
+      if (this.packageManager === "npm") {
+        this.npmInstall(typescriptDevDeps, { "save-dev": true })
```

Comment:
Yeoman is smart enough to only do one npm install, right? Even though we're calling npmInstall on line 148, 149, and 155?

Reply:
Yep, looks like it.

>  Yeoman will ensure the npm install command is only run once even if it is called multiple times by multiple generators.

from https://yeoman.io/authoring/dependencies.html